fedimint-wasm-tests/src/lib.rs

```diff
@@ -98,6 +99,16 @@ mod tests {
         Err(anyhow::anyhow!("Lightning receive failed"))
     }
 
+    #[wasm_bindgen_test]
+    async fn derive_chacha_key() -> Result<()> {
+        let root_secret = DerivableSecret::new_root(&[0x42; 32], &[0x2a; 32]);
+        let key = root_secret.to_chacha20_poly1305_key();
+        // Prevent optimization
+        println!("key: {key:?}");
```

Comment:
[`std::hint::black_box`](https://doc.rust-lang.org/stable/std/hint/fn.black_box.html)

because println might be noop on wasm.

Reply:
While at it, a comment explaining why we have it might be handy.

Reply:
@Maan2003 thx for the pointer, unfortunately it's only available on nightly :(

Reply:
I think our cargo is too old.

Reply:
black box has been stable since 1.66

Reply:
My cargo version inside `nix develop` is `cargo 1.67.0 (8ecd4f20a 2023-01-10)`

Reply:
https://doc.rust-lang.org/stable/src/core/hint.rs.html#290 says 1.66 though :thinking: